content/20.treatments.md

```diff
@@ -571,4 +571,37 @@ Summarize the state of the neutralizing antibody approach.
 
 ### Underexplored Therapeutics
 
-A summary of therapeutic avenues that future research may explore. Possible alternatives to current options and an overview of drug development processes.
+The majority of current clinical trials and lines of investigation have focused on repurposing existing therapies to counter SARS-CoV-2 and treat its symptoms.
+This is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
+However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
+There is thus value in investigating two lines of inquiry for treatment of COVID-19: 1) new therapeutics specific for treatment of COVID-19 or its symptoms, and 2) repurposing of existing therapeutics for treatment of COVID-19 or its symptoms.
+Here we consider further avenues that scientific investigators may explore in the development of therapies for COVID-19.
+
+Given the great focus in investigating hydroxychloroquine (HCQ) as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives.
+For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV [@doi:10.1021/jm0601856].
+Future work could explore whether these compounds exert antiviral effects against SARS-CoV-2 and if they are safe for use in animals and humans.
+
+The tocilizumab trial described in an above section [@doi:10/ggnxb3] studies the possibility of using an anti-inflammatory drug, typically used for the treatment of autoimmune disease, to counter the effects of the “cytokine storm” induced by the virus.
+Another anti-IL-6 antibody, sarilumab, is also being investigated [how to cite? http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19 or https://clinicaltrials.gov/ct2/show/NCT04327388].
```

Comment:
This is great ! I think it would be awesome to write a couple of sentences about tocilizumab's costs at this point. Because even if tocilizumab is effective, not everyone is going to be able to afford it and production is slow. Cyclosporin is cheap and readily available and might have similar effects. This might be worth discussing

Reply:
I think this is a very important addition, thanks for the suggestion! I just added in a few lines about it, let me know what you think.